src/routes/safe/components/Transactions/TxsTable/ExpandedTx/TxDescription/Value.tsx

```diff
@@ -25,56 +19,41 @@ const StyledText = styled(Text)`
 interface RenderValueProps {
   method: string
   type: string
-  value: string | string[]
+  value: string | unknown[]
```

Comment:
what was a problem there? by looking at the usage it seems the param is always string

Reply:
fixed